### DISCUSSION TITLE
api文档中有获取某个namespace release 所有历史版本api吗

### DISCUSSION THREAD

**Discussion by yang0009** [apolloconfig/apollo#3722] [Q&A] [answered]
api文档中有获取某个namespace release 所有历史版本api吗

**Comment by nobodyiam** [comment, marked as answer]
open api 中没有这个接口，在 portal api 中是有的（com.ctrip.framework.apollo.portal.controller.ReleaseController#findActiveReleases）

**Reply by yang0009** [reply]
好的,谢谢
